api/envoy/config/filter/http/grpc_stats/v2alpha1/config.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.grpc_stats.v2alpha1;
```

Comment:
We don't do `*alpha*1` anymore. Just `v2alpha`. I think @htuch was going to block this with a format rule but it hasn't been done yet.

Reply:
Updated.